src/main/resources/reference.conf

```diff
@@ -36,4 +40,6 @@ userprofile {
 
 googlecloud {
   priceListUrl: "https://cloudpricingcalculator.appspot.com/static/data/pricelist.json"
-}
\ No newline at end of file
+}
+
+spray.can.client.response-chunk-aggregation-limit = 5m
```

Comment:
based on the release-debugging I just did, this needs to move out of reference.conf and into firecloud-orchestration.conf, so it won't get clobbered by spray defaults. Note that our other spray config overrides are in firecloud-orchestration.conf too.

The way to test this is to call `sbt assembly` and then run the resultant jar, instead of running from the sbt shell.